components/docs-chef-io/content/automate/ha_existing_a2ha_to_automate_ha.md

```diff
@@ -204,6 +228,41 @@ In Automate HA there are equivalent command which had been used in A2HA:
 | gather logs                | `bash automate-cluster-clt gather-logs `                  | `bash chef-automate gather-logs`                              |
 | workspace                  | `bash automate-cluster-clt workspace `                    | `bash chef-automate workspace [OPTIONS] SUBCOMMAND [ARG] ...` |
 
+## Validate successful migration
+
+1. Check the Automate UI of Automate HA. Check whether the data is present in Automate UI for HA.
+1. If you are using the embedded chef server, log in to the Chef Server HA node, and run the following script to get a count of objects from the Chef Infra Server, this should match the counts captured at the start of the migration
```

Comment:
Replace it with: 1. If you use the embedded chef server, log in to the Chef Server HA node, and run the following script to get a count of objects from the Chef Infra Server. This should match the counts captured at the start of the migration